README.md

```diff
@@ -26,7 +26,7 @@ Check out the article for more info: [paulmillr.com/posts/encrypted-dns/](https:
 | Cloudflare                | 🇺🇸      | No          | [Operated](https://developers.cloudflare.com/1.1.1.1/dns-over-https) by Cloudflare 1.1.1.1                                              | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/cloudflare-https.mobileconfig), [TLS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/cloudflare-tls.mobileconfig)                             |
 | Cloudflare Malware        | 🇺🇸      | Yes          | Filters malware                                                                                                                         | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/cloudflare-malware-https.mobileconfig)                                                                                                                |
 | Cloudflare Family         | 🇺🇸      | Yes          | Filters malware & adult content                                                                                                         | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/cloudflare-family-https.mobileconfig)                                                                                                                 |
-| DNSPod                    | 🇨🇳      | Yes          | [Operated](https://docs.dnspod.cn/public-dns/5fb5db1462110a2b153a77dd/) in mainland China                                               | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/dnspod-https.mobileconfig),  [TLS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/dnspod-tls.mobileconfig)                                                                                                                            |
+| DNSPod                    | 🇨🇳      | Yes          | [Operated](https://dns.pub/?lang=en) by Tencent in Mainland China                                               | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/dnspod-https.mobileconfig),  [TLS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/dnspod-tls.mobileconfig)                                                                                                                            |
```

Comment:
can you change this to https://www.dnspod.cn/Products/publicdns?lang=en

Reply:
all set